openshift-data-foundation-operator/operator/base/enable-console-plugin-job.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: enable-odf-console-plugin
+  generateName: enable-odf-console-plugin-
+  annotations:
+    argocd.argoproj.io/sync-wave: "10"
```

Comment:
I'm not sure sync-wave should be specified. Rather let the consumer of this specify if required downstream.

Reply:
I was getting an odd issue where if the RBAC and SA is applied at the same time as the Job the pod would get an `imagePullBackOff` error.  That error prevents the job from attempting to automatically retry so the job gets stuck and you have to manually delete the pod.  Once the pod is deleted it creates a new one and pulls the image successfully.  

Adding the sync-wave prevented the issue from occurring and gave time for whatever internal reconciliation needs to happen with the RBAC/SA to sort itself out.

I'm happy to remove it but I expect others may run into similar issues.

Reply:
Yes, it's odd, but not a big deal from my standpoint. I'm use to put the RBAC and job in the same file, maybe this is why I don't need SyncWave because Argo walk the objects in the file in parallel. I can't tell.
Anyhow, if this recipe is what works, then LGTM.
Thanks

Reply:
Yes, it's odd, but not a big deal from my standpoint. I'm use to put the RBAC and job in the same file, maybe this is why I don't need SyncWave because Argo walk the objects in the file in parallel. I can't tell.
Anyhow, if this recipe is what works, then LGTM.
Thanks

Reply:
@strangiato @adetalhouet , it sounds like you guys have looked into this deeper than I have.  If you guys feel this is ready to merge, I'll merge it :)